mne/time_frequency/psd.py

```diff
@@ -168,7 +168,7 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
     else:
         time_mask = Ellipsis
 
-    data = epochs.get_data()[:, picks][..., time_mask]
+    data = epochs.get_data()[:, picks][:, :, time_mask]
```

Comment:
Debian branch should have no modifications in the upstream code. Those should be absorbed into patches